prettier.config.js

```diff
@@ -13,4 +13,6 @@
  * limitations under the License.
  */
 
-module.exports = require('@sumup/foundry/prettier')();
+module.exports = require('@sumup/foundry/prettier')(undefined, {
+  proseWrap: 'always',
```

Comment:
As mentioned in the commit message body, this will make writing and reviewing docs easier for us moving forward

Reply:
How so? I would think that it does the exact opposite 🤔

Reply:
My initial thought was that wrapping lines made it easier to work with in editors (no need to set up wrapping) and for suggesting changes on a line, since they'll be of a manageable length. But on second thought, this might indeed make it harder, because we won't be able to apply changes directly from the GH UI (need to format locally), and so on. I'll revert the change

Reply:
Done in 0e33e73

Reply:
Well, hard-coding the line-wraps in combination with editor soft-wraps would lead to ragged lines. 

Thanks for reverting the change! 🙇🏻‍♂️